src/games/strategy/triplea/ui/MapData.java

```diff
@@ -204,11 +204,23 @@ private void loadTerritoryNames() {
     }
     for (final String name : m_centers.keySet()) {
       try {
-        final Image img = loadImage("territoryNames/" + name + ".png");
+        String normalizedName = name.replace(' ', '_');
+        String path = "territoryNames/" + normalizedName + ".png";
+        Image img = loadImage(path);
+
+        if(img == null ) {
+          path = "territoryNames/" + normalizedName + ".png";
```

Comment:
Yes, I realized that later though when I was refactoring this class. Originally:
#900 was a straight bug fix
#901 (this PR) was a move
#902 is a PR into #901 - refactoring MapData

I closed #900 since the fix was not right there.  #902 still contains the fix: 3c9282159b1ca173f29f4f2605bf20a73ae3fa49